static/js/autosave.ts

```diff
@@ -34,18 +28,6 @@ export function autoSave(formId: string,
         }
           // Now we can simply trigger the submit event on the parent form element.
           formElement.requestSubmit()
-
-          // Sometimes different components depend on each other, that is component A needs to be updated if component B
```

Comment:
This is probably me not understanding how this works. But why was this code added, i.e. why did we have to update partial-sortable-adventures when customize-class was updated? And why are we removing it?